src/bin/sus-kernel/permission/verify/sudoers.rs

```diff
@@ -0,0 +1,54 @@
+use super::sudoers_type::Sudoers;
+use super::{Verifier, VerifyError};
+use crate::permission::verify::VerifyResult;
+use nix::unistd::{Gid, Uid};
+use std::ffi::CString;
+use std::fs::File;
+use std::io::BufReader;
+
+#[allow(dead_code)]
+#[derive(Debug)]
+struct Command {
+    run_as_users: Vec<Uid>,
+    run_as_groups: Vec<Gid>,
+    commands: Vec<CString>,
+}
+
+#[derive(Debug)]
+struct Policy {
+    users: Vec<Uid>,
+    groups: Vec<Gid>,
+    cmd_specs: Vec<Command>,
+}
+
+#[allow(dead_code)]
+pub fn from_sudoers() -> Vec<Box<Verifier>> {
+    // Declare vector of verifiers to return
+    let mut verifiers = Vec::new();
+    // Parse sudoers.json using serde_json
+    let file = File::open("sudoers.json").unwrap();
```

Comment:
It might be a bad idea to read from a hard-coded file name, especially one that is in the current user's directory. Maybe have the path be in the configuration file.